### ISSUE TITLE
options does not disappear, if clicking an already selected option.

### ISSUE THREAD

**Issue by Jonathhhan** [roymacdonald/ofxDropdown#23]
It disappears only after the second click. Maybe the best behavior would be, if the request gets cancelled and the options disappear...

**Comment by Jonathhhan** [comment]
Solved it in this pr: https://github.com/roymacdonald/ofxDropdown/pull/21

**Comment by roymacdonald** [comment]
right. It would be better if it would closed the dropdown, otherwise it leaves it in a weird state where nothing is selected.
